src/Facebook/Auth.hs

```diff
@@ -60,12 +60,12 @@ import Facebook.Monad
 -- Ref: https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow
 getAppAccessToken
   :: (R.MonadResource m, R.MonadUnliftIO m, R.MonadThrow m)
-  => FacebookT Auth m AppAccessToken
-getAppAccessToken =
+  => Text -> FacebookT Auth m AppAccessToken
```

Comment:
Can you add Haddock documentation on what this `Text` it is ?

Reply:
Also, if you can link to the facebook's API documentation on where to find this info it will be helpful.